environment.yml

```diff
@@ -0,0 +1,6 @@
+name: pygmt-workshop
+channels:
+    - conda-forge
+dependencies:
+    - pygmt
+    - jupyterlab
```

Comment:
I got a strange error when importing `pygmt` using this environment. Something with xarray perhaps?

<details>

```python-traceback
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-1-f58669fae37f> in <module>
----> 1 import pygmt

/srv/conda/envs/notebook/lib/python3.7/site-packages/pygmt/__init__.py in <module>
     24 
     25 # Import modules to make the high-level GMT Python API
---> 26 from pygmt import datasets
     27 from pygmt.accessors import GMTDataArrayAccessor
     28 from pygmt.figure import Figure, set_display

/srv/conda/envs/notebook/lib/python3.7/site-packages/pygmt/datasets/__init__.py in <module>
      3 # Load sample data included with GMT (downloaded from the GMT cache server).
      4 
----> 5 from pygmt.datasets.earth_relief import load_earth_relief
      6 from pygmt.datasets.samples import (
      7     load_fractures_compilation,

/srv/conda/envs/notebook/lib/python3.7/site-packages/pygmt/datasets/earth_relief.py in <module>
      5 The grids are available in various resolutions.
      6 """
----> 7 import xarray as xr
      8 from pygmt.exceptions import GMTInvalidInput
      9 from pygmt.helpers import kwargs_to_strings

/srv/conda/envs/notebook/lib/python3.7/site-packages/xarray/__init__.py in <module>
      1 import pkg_resources
      2 
----> 3 from . import testing, tutorial, ufuncs
      4 from .backends.api import (
      5     load_dataarray,

/srv/conda/envs/notebook/lib/python3.7/site-packages/xarray/testing.py in <module>
      6 import numpy as np
      7 
----> 8 from xarray.core import duck_array_ops, formatting, utils
      9 from xarray.core.dataarray import DataArray
     10 from xarray.core.dataset import Dataset

/srv/conda/envs/notebook/lib/python3.7/site-packages/xarray/core/duck_array_ops.py in <module>
     13 import pandas as pd
     14 
---> 15 from . import dask_array_compat, dask_array_ops, dtypes, npcompat, nputils
     16 from .nputils import nanfirst, nanlast
     17 from .pycompat import (

/srv/conda/envs/notebook/lib/python3.7/site-packages/xarray/core/npcompat.py in <module>
     37 # Type annotations stubs
     38 try:
---> 39     from numpy.typing import ArrayLike, DTypeLike
     40 except ImportError:
     41     # fall back for numpy < 1.20, ArrayLike adapted from numpy.typing._array_like

/srv/conda/envs/notebook/lib/python3.7/site-packages/numpy/typing/__init__.py in <module>
    314 )
    315 from ._shape import _Shape, _ShapeLike
--> 316 from ._dtype_like import (
    317     DTypeLike as DTypeLike,
    318     _SupportsDType,

/srv/conda/envs/notebook/lib/python3.7/site-packages/numpy/typing/_dtype_like.py in <module>
     93     _DTypeDict = Any
     94 
---> 95     class _SupportsDType(Generic[_DType_co]):
     96         pass
     97 

NameError: name '_DType_co' is not defined
```
</details>